src/main/java/seedu/contax/model/appointment/AppointmentSlot.java

```diff
@@ -4,7 +4,8 @@
 import seedu.contax.model.chrono.TimeRange;
 
 /**
- * Represents an empty slot in the schedule.
+ * Represents an empty slot in the schedule. Time related functionality is handled in the superclass
```

Comment:
```suggestion
 * Represents an empty slot in the schedule. Time related functionality is handled in the superclass.
```

Reply:
Likewise, the sentence continues to the next line for the linking, which is why there is no fullstop here.